### ISSUE TITLE
Improve Collection creator UI

### ISSUE THREAD

**Issue by JJ-Author** [dbpedia/databus-maven-plugin#112]
- [ ] include the new search type filter component from #110 when adding a new item

collection group facet configuration
- [ ] show name of the group which is configured
- [ ] render the list of artifacts which are selected for the group being edited 
- [ ] show the number of files per artifact which are returned based on (should be fetched async in order to not slow down UI experience)  group facet selection vs overwritten facet selection (if any)??

collection artifact facet configuration
- [ ] show name of the artifact which is configured

**Comment by Vehnem** [comment]
- [ ] Select latest version as default

**Comment by holycrab13** [comment]
I am not sure about the remaining suggestions, this would do a LOT of querying in the background.

**Comment by JJ-Author** [comment]
yeah I agree. The new experience is great. But when you click on a group in the collection you get this list with artifacts and the group filter selection. I think at this place it could make sense to give this information, how many files are selected by group and how many are overriden? but potentially you are right an this will trigger a lot of queries. maybe sth for backlog?
